rasa/constants.py

```diff
@@ -59,3 +59,6 @@
 
 DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES = 60
 DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION = True
+
+DEFAULT_SENDER_ID = "default"
+
```

Comment:
@VictorCoCo now there are two blank lines, again leading to failing tests 😄 
```suggestion
```

Reply:
Should be fixed now :)